accelerometer/LinearAccelerationSensor-threshold-manual.https.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>LinearAccelerationSensor Shake Threshold Test</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="https://w3c.github.io/accelerometer/#examples">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<p class="instructions">
+  Shake gesture along x axis of the device.
+</p>
+<script>
+
+setup({explicit_timeout: true});
+
+async_test(t => {
+  const shakeThreshold = 25;
+  const sensor = new LinearAccelerationSensor({frequency: 60});
+
+  sensor.onreading = t.step_func(() => {
+    if (sensor.x > shakeThreshold) {
+      t.done();
+    }
+  });
+
+  sensor.onerror = t.step_func_done(event => {
+    assert_unreached(event.error.name + ":" + event.error.message);
+  });
+
+  sensor.start();
+}, "Test that shake gesture along x axis of the device, threshold can greater than 25");
```

Comment:
's/shake/when shaking'
's/threshold/the shake threshold'
's/can/can be'

Reply:
Fixed, @Honry, PTAL.